modification de la charte et des reglements faisant suite à l'AG du 16 septembre 2024

Overengined opened the pull request:
Suite à l'assemblée générale du 16 septembre, les changements proposés par le "Projet de modification à la Charte de l’AÉDIROUM" ont été exécutés. Rencontré des difficultés lors de l'exécution de la modification nº49, il est possible que certaines références a des articles de la charte aient été mis a jour de façon incorrecte ou insatisfaisante, n'hésitez pas a me les signaler.

Overengined committed:
modification faisant suite à la modification de l'AG du 16 septembre 2024

Suite à l'assemblée générale du 16 septembre, les changements proposés par le "Projet de modification à la Charte de l’AÉDIROUM" ont été exécutés.
Rencontré des difficultés lors de l'exécution de la modification nº49, il est possible que certaines références a des articles de la charte aient été mis a jour de façon incorrecte ou insatisfaisante, n'hésitez pas a me les signaler.

matteodelabre commented on charte.tex:
```diff
@@ -13,7 +13,8 @@ \section{Généralités}\label{sec:generalites}
     \item[C.A.] Conseil d’administration
-    \item[C.R.] Comité de représentation
+    \item[C.E] Conseil exécutif
+    \item[C.R.] Comité de représentation de l’AÉDIROUM, tel que défini à l’article 4 de la Charte de l’AÉDIROUM
```
Devrait être simplement
```suggestion
    \item[C.R.] Comité de représentation
```

matteodelabre commented on charte.tex:
```diff
@@ -175,9 +198,29 @@ \subsection{Fonctions du C.A.}\label{sec:fonctions-du-conseil-executif}
+Le Comité de représentation~(C.R.) est composé des douze~(12) personnes élues de
+l’AÉDIROUM aux postes suivants :
 \begin{itemize}
 \item Représentation de première année au baccalauréat en informatique
```
Amendement no. 14: Insérer «Vice-présidence aux affaires académiques» au début de la liste

Overengined replied:
résolu dans 810f18d

matteodelabre commented on charte.tex:
```diff
@@ -189,7 +232,7 @@ \subsection{Composition du C.R.}\label{sec:composition-du-conseil-dadministratio
 \item Représentation des personnes étudiantes internationales
 \item Représentation du Regroupement de Femmes en Informatique
-\item Toute personne adjointe à un poste du C.A.
+\item Représentation des personnes étudiantes du Mila
```
Devrait être
```suggestion
\item Représentation des personnes étudiantes de Mila
```

matteodelabre commented on charte.tex:
```diff
@@ -202,11 +245,11 @@ \subsubsection{Représentation des personnes étudiantes internationales}\label{
 \subsubsection{Représentation du café Tore et Fraction}\label{sec:representant-du-cafe-etudiant-math-info}
 
-La tâche de la personne représentante du café est définie dans la charte du café. Elle doit former un comité constitué de personnes membres de l'AÉDIROUM pour remplir les postes requis par la charte du café. Elle représente l'administration du café devant l'association et le C.R. devant l'administration du café.
+La tâche de la personne représentante du café est définie dans la Charte du café. Elle doit former un comité constitué de personnes membres de l'AÉDIROUM pour remplir les postes requis par la Charte du café. Elle représente l'administration du café devant l'association et le C.R. devant l'administration du café.
```
Doivent rester en minuscule car ne réfèrent pas à la Charte de l’AÉDIROUM
```suggestion
La tâche de la personne représentante du café est définie dans la charte du café. Elle doit former un comité constitué de personnes membres de l'AÉDIROUM pour remplir les postes requis par la charte du café. Elle représente l'administration du café devant l'association et le C.R. devant l'administration du café.
```

matteodelabre commented on charte.tex:
```diff
@@ -218,25 +261,34 @@ \subsubsection{Représentation du Regroupement de Femmes en Informatique}\label{
-Doit être recommandée par le RFIUM avant de postuler à L'AÉDIROUM, avec approbation de 50\%+1 vote, lors d'une AG ou CA du RFIUM ayant eu lieu préalablement, soit durant la même session ou durant la session précédant celle-ci.  Le RFIUM doit avoir élu au moins cinq~(5) personnes membres de son conseil administratif au moment de l'élection.
+La personne représentante doit être recommandée par le
+RFIUM avant de postuler à l’AÉDIROUM, lors d’une assemblée générale ou d’une réunion du conseil d’administration du RFIUM ayant eu lieu préalablement, soit durant la même session ou durant la session précédant celle-ci. Elle doit obtenir, lors d’une telle
+assemblée ou réunion, l’approbation d’au moins 50% des voix plus une voix. Le RFIUM doit avoir élu au moins cinq (5) personnes membres de son conseil d’administration au moment de l’élection.
```
Le pourcentage doit être échappé
```suggestion
assemblée ou réunion, l’approbation d’au moins 50\% des voix plus une voix. Le RFIUM doit avoir élu au moins cinq (5) personnes membres de son conseil d’administration au moment de l’élection.
```

matteodelabre commented on charte.tex:
```diff
@@ -218,25 +261,34 @@ \subsubsection{Représentation du Regroupement de Femmes en Informatique}\label{
+assemblée ou réunion, l’approbation d’au moins 50% des voix plus une voix. Le RFIUM doit avoir élu au moins cinq (5) personnes membres de son conseil d’administration au moment de l’élection.
+
+\subsubsection{Représentation des personnes
+étudiantes de Mila}
```
```suggestion
\subsubsection{Représentation des personnes étudiantes de Mila}
```

matteodelabre commented on charte.tex:
```diff
@@ -218,25 +261,34 @@ \subsubsection{Représentation du Regroupement de Femmes en Informatique}\label{
 
 \paragraph{Conditions d'éligibilité}
 
-Doit être recommandée par le RFIUM avant de postuler à L'AÉDIROUM, avec approbation de 50\%+1 vote, lors d'une AG ou CA du RFIUM ayant eu lieu préalablement, soit durant la même session ou durant la session précédant celle-ci.  Le RFIUM doit avoir élu au moins cinq~(5) personnes membres de son conseil administratif au moment de l'élection.
+La personne représentante doit être recommandée par le
+RFIUM avant de postuler à l’AÉDIROUM, lors d’une assemblée générale ou d’une réunion du conseil d’administration du RFIUM ayant eu lieu préalablement, soit durant la même session ou durant la session précédant celle-ci. Elle doit obtenir, lors d’une telle
+assemblée ou réunion, l’approbation d’au moins 50% des voix plus une voix. Le RFIUM doit avoir élu au moins cinq (5) personnes membres de son conseil d’administration au moment de l’élection.
+
+\subsubsection{Représentation des personnes
+étudiantes de Mila}
+La personne représentante des personnes étudiantes de Mila est responsable de contacter,
+en français et en anglais, les personnes membres qui sont également membres de
+l’Institut québécois d’intelligence artificielle (Mila). Elle doit rejoindre et mobiliser les
+personnes membres pour prendre part aux actions et aux activités de l’AÉDIROUM. Elle
+représente l’AÉDIROUM au sein de Mila et Mila au sein de l’AÉDIROUM. Seules les
+personnes à la fois membres de l’AÉDIROUM et de Mila sont éligibles pour ce poste.
 
 \subsubsection{Cumul de postes}\label{sec:cumul-de-postes}
 
 Sans aller à l'encontre de l'\article{sec:composition-du-conseil-executif}, il peut y avoir, au sein du C.R., cumul de postes par une même personne. Au sein de l'\article{sec:quorum-et-participation}, le cumul de poste n'entraine pas de voix supplémentaire lors d'un vote en réunion du C.R.
 
-\subsection{Autres membres du comité du café Tore et Fraction}\label{sec:autres-membres-du-comite-du-cafe-etudiant}
-
-Seront élues deux autres personnes membres du comité du café étudiant, qui ne seront pas membres du C.R. L'un de ces postes sera ouvert en priorité à une personne étudiante de première année afin de favoriser le bon roulement des personnes membres du comité du café. C'est ce poste pour lequel l'élection se fera en automne. Parmi le représentation du café étudiant et les deux personnes membres élu ...
-
-\subsection{Personnes adjointes}\label{sec:adjoints}
+\subsubsection{Procédure de preuve}\label{sec:procedure-de-preuve}
 
-Une personne adjointe peut être élue en assemblée générale pour chaque poste du C.A. défini à l'\article{sec:composition-du-conseil-executif}.
+Afin qu'une personne membre du C.R., sa succession et ses ayants droits puissent se prévaloir des dispositions d'indemnisation, le C.A. peut exiger des preuves suffisantes que les frais, charges ou dépenses visées à l'\article{sec:indemnisation} ont subies.
 
```
Est-ce que cette sous-section est à la bonne place?

Overengined replied:
Je ne crois pas qu'il soit fait mention dans le projet de modification de déplacer cette sous section à un endroit particulier. Tu trouve qu'elle ne fait de sens dans le contexte ?

matteodelabre commented on charte.tex:
```diff
@@ -218,25 +261,34 @@ \subsubsection{Représentation du Regroupement de Femmes en Informatique}\label{
+Le Conseil exécutif~(C.E.) est composé des membres du C.A. et du C.R. et des personnes
+adjointes à des membres du C.A.
 
 \subsection{Réunions du C.R.}\label{sec:reunions-du-conseil-dadministration}
```
```suggestion
\subsection{Réunions du C.E.}\label{sec:reunions-du-conseil-dadministration}
```

matteodelabre commented on charte.tex:
```diff
@@ -257,7 +309,7 @@ \subsubsection{Quorum et participation}\label{sec:quorum-et-participation}
 \subsubsection{Procédure}\label{sec:procedure-admin}
 
-Le C.R. peut adopter tout règlement concernant sa procédure d'assemblée, dans la mesure où ce règlement ne contrevient pas à la charte.
+Le C.R. peut adopter tout règlement concernant sa procédure d'assemblée, dans la mesure où ce règlement ne contrevient pas à la Charte.
```
```suggestion
Le C.E. peut adopter tout règlement concernant sa procédure d'assemblée, dans la mesure où ce règlement ne contrevient pas à la Charte.
```

matteodelabre commented on charte.tex:
```diff
@@ -304,18 +356,6 @@ \subsection{Rémunération}\label{sec:renumeration}
-
 \subsection{Intérim}\label{sec:interim}
 
 Le C.R. peut, par résolution, nommer par intérim toute personne membre de l'AÉDIROUM éligible à un poste vacant du C.R., y compris à un poste d'administratrice. Tout poste dont la personne titulaire a été ainsi nommée est mis en élection à l'assemblée générale régulière qui suit.
```
```suggestion
Le C.E. peut, par résolution, nommer par intérim toute personne membre de l'AÉDIROUM éligible à un poste vacant du C.E., y compris à un poste d'administratrice. Tout poste dont la personne titulaire a été ainsi nommée est mis en élection à l'assemblée générale régulière qui suit.
```

matteodelabre commented on charte.tex:
```diff
@@ -304,18 +356,6 @@ \subsection{Rémunération}\label{sec:renumeration}
 
 À moins qu'une telle mesure ne soit votée en assemblée générale, notamment lors d'un vote sur le budget, aucune des personnes membres du C.R. n'est rémunérée pour l'exercice de ses fonctions. Elles ne peuvent pas non plus tirer profit financier d'activités faites au nom de l'AÉDIROUM.
```
```suggestion
À moins qu'une telle mesure ne soit votée en assemblée générale, notamment lors d'un vote sur le budget, aucune des personnes membres du C.E. n'est rémunérée pour l'exercice de ses fonctions. Elles ne peuvent pas non plus tirer profit financier d'activités faites au nom de l'AÉDIROUM.
```

matteodelabre commented on charte.tex:
```diff
@@ -328,15 +368,20 @@ \section{Comités}\label{sec:comites}
 
 Le C.R. ou les personnes membres réunies en assemblée peuvent mettre sur pied tout comité jugé utile à la bonne marche de l'AÉDIROUM.
```
```suggestion
Le C.E. ou les personnes membres réunies en assemblée peuvent mettre sur pied tout comité jugé utile à la bonne marche de l'AÉDIROUM.
```

matteodelabre commented on charte.tex:
```diff
@@ -328,15 +368,20 @@ \section{Comités}\label{sec:comites}
+Lors de la mise sur pied du comité, la nature de son mandat ainsi que sa durée doivent être spécifiées. Si le comité est permanent, ses missions et le nom des postes élus qui sont appelés à le constituer doivent être définis dans la présente Charte ou dans un règlement.
 
-Une présidence doit aussi être nommée et choisie par le C.R. parmi les personnes membres de l'AÉDIROUM. À la demande du C.R., la présidence d'un comité doit faire un rapport oral au C.R. À la fin du mandat, les activités du comité doivent cesser et la présidence du comité doit alors présenter, lors de la prochaine assemblée générale des personnes membres, un rapport écrit du bilan du comité.
+Une présidence doit aussi être nommée et choisie par le C.R. parmi les personnes membres de l'AÉDIROUM. À la demande du C.R., la présidence d'un comité doit faire un rapport oral au C.R. À la fin du mandat, les activités de tout comité non-permanent
```
```suggestion
Une présidence doit aussi être nommée et choisie par le C.E. parmi les personnes membres de l'AÉDIROUM. À la demande du C.E., la présidence d'un comité doit faire un rapport oral au C.E. À la fin du mandat, les activités de tout comité non-permanent
```

matteodelabre commented on charte.tex:
```diff
@@ -328,15 +368,20 @@ \section{Comités}\label{sec:comites}
+doivent cesser et la présidence du comité doit alors présenter, lors de la prochaine assemblée générale des personnes membres, un rapport écrit du bilan du comité.
 
-\section{Modification à la charte}\label{sec:modification-a-la-charte}
+\section{Modification à la Charte}\label{sec:modification-a-la-Charte}
```
Laisser le label tel quel
```suggestion
\section{Modification à la Charte}\label{sec:modification-a-la-charte}
```

matteodelabre commented on charte.tex:
```diff
@@ -328,15 +368,20 @@ \section{Comités}\label{sec:comites}
+\section{Modification à la Charte}\label{sec:modification-a-la-Charte}
 
-Tout amendement à la présente charte doit être proposé lors d'une assemblée générale et adopté aux deux tiers (\( \frac{2}{3} \)) (arrondi vers le haut) des voix exprimées. Tout changement substantiel doit être rendu disponible dans l'avis de convocation.
+Tout changement substantiel doit être rendu consultable par les membres au moins
```
Seule la première phrase du premier paragraphe de la version précédente doit être remplacé («Tout changement substantiel doit être rendu disponible dans l'avis de convocation.»). Le reste de l’article est inchangé.

Overengined replied:
Normalement, résolu dans 810f18d . C'est bien ce qui n'allait pas ?

matteodelabre commented on coussin.tex:
```diff
@@ -25,26 +25,19 @@ \subsection{Définitions}\label{sec:definitions}
 \begin{description}
     \item[AÉDIROUM] Association Étudiante du Département d'Informatique et de Recherche Opérationnelle de l'Université de Montréal
     \item[C.A.] Conseil d’administration
```
```suggestion
    \item[C.A.] Conseil d’administration de l’AÉDIROUM, tel que défini à l’article~3 de la Charte
```

matteodelabre commented on coussin.tex:
```diff
@@ -75,34 +68,48 @@ \subsection{Le COUSSIN}\label{sec:coussin}
 La personne responsable des MiDiro assure la planification, l'organisation et la tenue des conférences MiDiro.
 
-\subsection{Responsable Compétition}\label{sec:responsable-competition}
+\subsection{Responsable des Compétition}\label{sec:responsable-competition}
```
```suggestion
\subsection{Responsable des compétitions}\label{sec:responsable-competition}
```

matteodelabre commented on coussin.tex:
```diff
@@ -75,34 +68,48 @@ \subsection{Le COUSSIN}\label{sec:coussin}
-
-\subsection{Responsable Sport}\label{sec:responsable-sport}
-La personne responsable sport s'occupe d'organiser des activités sportives amicales ainsi que de la sélection et la participation officielle de l'AÉDIROUM à diverses activités sportives.
+\subsection{Responsable des Sports}\label{sec:responsable-sport}
```
```suggestion
\subsection{Responsable des sports}\label{sec:responsable-sport}
```

matteodelabre commented on coussin.tex:
```diff
@@ -198,7 +205,7 @@ \subsection{Les MiDiros}\label{sec:midiros}
 		\item Informer les personnes étudiantes du 1er cycle sur différents aspects des études supérieures.
 	\end{itemize}
-	\item[Responsables suggérés] La personne responsable MiDiro.
+	\item[Responsables suggérés] La personne Responsable des MiDiro.
```
```suggestion
	\item[Responsables suggérés] La personne responsable des MiDiro.
```

matteodelabre commented on faecum.tex:
```diff
@@ -17,30 +17,28 @@ \subsection{Définitions}\label{sec:definitions}
+cahier de positions de l’association, établi par l’article 10 de la Charte.
 
-\subsection{Délégués d'office aux instances et sous-instances}\label{sec:delegues-doffice-instances}
+\subsection{Personnes déléguées
+d’office aux instances et sous-instances}\label{sec:delegues-doffice-instances}
```
```suggestion
\subsection{Personnes déléguées d’office aux instances et sous-instances}\label{sec:delegues-doffice-instances}
```

matteodelabre reviewed (commented):
Merci Théodore! Voici quelques rectifications.

matteodelabre commented:
Il faut aussi mettre à jour la date en ligne 6 à la date de l’AG.

Overengined committed:
mod matteo 1

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod matteo 2

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod matteo 3

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 4

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 5

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 6

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 7

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 8

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 9

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 10

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 11

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 12

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 13

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 14

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 15

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 16

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
mod Mattéo 17

Co-authored-by: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

Overengined committed:
date ligne 6

Overengined committed:
corrections liste rep et procedure modif charte

ajouté Vice-présidence aux affaires académiques
corrigé la procédure de modification de la charte

Co-Authored-By: Mattéo Delabre <1370040+matteodelabre@users.noreply.github.com>

matteodelabre committed:
Corrige prise en charge des apostrophes dans PDF

matteodelabre committed:
Corrige typo dans «Personnes adjointes»

matteodelabre committed:
Replace «Procédure de preuve» dans la bonne sous-section

matteodelabre committed:
Corrige références au C.R. → C.E.

matteodelabre committed:
Consistance des délimiteurs de mode math

matteodelabre committed:
Sept personnes signataires du compte de banque

matteodelabre committed:
Corrections de style RFIUM

matteodelabre committed:
aediroum.ca en monospace

matteodelabre committed:
Dates de modification COUSSIN et FAÉCUM

matteodelabre committed:
Typo responsable compétitions

matteodelabre committed:
Corrections de style FAÉCUM

matteodelabre commented:
J’ai fait quelques corrections supplémentaires. Je te laisse vérifier que tout est bon @Overengined!

matteodelabre committed:
S’assure d’avoir un nouveau paragraphe au début de chaque section pour make4ht
